index.js

```diff
@@ -0,0 +1,22 @@
+const axios = require("axios").default;
+const player = require('play-sound')();
+const api = axios.get(
+  "https://heb-ecom-covid-vaccine.hebdigital-prd.com/vaccine_locations.json"
+);
+
+var time = .1, interval = time*60*1000;
+setInterval(() => {
```

Comment:
podemos convertir estos a async/await 

```suggestion
setInterval(async () => {
```